### DISCUSSION TITLE
URL to home view currently playing

### DISCUSSION THREAD

**Discussion by Hassi1** [music-assistant/support#2295] [Feature requests and ideas]
Hi, 

I really love this project. I was wondering if it is possible to add support for a direct navigation to home view with the currently playing container in full screen. I have a tap action that navigates to music assistant home view but would like it to pop up the currently playing container in full screen.

Thank you 🙏

**Comment by OzGav** [comment]
I think this covers this request https://music-assistant.io/faq/how-to/#access-the-now-playing-view-directly-via-url

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
